database-seeds/collections/charts-iidx.json

```diff
@@ -9234,14 +9234,14 @@
 			"hcTier": {
 				"individualDifference": false,
 				"text": "11S",
-				"value": 12
```

Comment:
It looks like all decimal ratings went down - was this intended? (not really sure how the letter tiers map to the decimal ratings...)

Reply:
This is part of what I want feedback on.  I have no idea how these values were decided on previously.  I see charts in the current data that are in the same "letter" tier but with different numeric values, I don't know if those were conscious decisions or not.

I'm using this mapping

		case 'S+': return level + .9;
		case 'S': return level + .8;
		case 'A+': return level + .7;
		case 'A': return level + .6;
		case 'B+': return level + .5;
		case 'B': return level + .4;
		case 'C': return level + .3;
		case 'D': return level + .2;
		case 'E': return level + .1;
		case 'F': return level + .0;
		default: return level;